### PR TITLE
Clean up build configurations

### PR DESCRIPTION
**Pull request by blindpirate**
1. Remove file leak detector in Windows builds. We haven't seen such
   issues for a long time.
2. Remove AdoptiumOpenJDK as vendor because we will only have two
   vendors (Oracle/Temurin) in the future.

### CODE REVIEW

**Comment by blindpirate** [comment]
@bot-gradle test this

**Comment by blindpirate** [comment]
@bot-gradle test this

File: .teamcity/src/main/kotlin/common/Jvm.kt [hunk 1]
```diff
@@ -30,5 +30,5 @@ object BuildToolBuildJvm : Jvm {
     override val version: JvmVersion
         get() = JvmVersion.java11
     override val vendor: JvmVendor
-        get() = JvmVendor.adoptiumopenjdk
+        get() = JvmVendor.openjdk
```
Comment (wolfs) [inline .teamcity/src/main/kotlin/common/Jvm.kt:33]: Does that mean that Temurin returns `OpenJdk`?

Comment (blindpirate) [reply .teamcity/src/main/kotlin/common/Jvm.kt:33]: Because now `linux.javaX.openjdk.64bit` points to temurin now, so this change is fine.

**Comment by blindpirate** [comment]
@bot-gradle test and merge

**Pull request merged**